console/src/main/scripts/plugins/metrics/ts/app-details/appServerJvmDetails.ts

```diff
@@ -143,21 +143,57 @@ module HawkularMetrics {
     }
 
     private formatCounterChartOutput(response, buckets = 60):IChartDataPoint[] {
```

Comment:
Can this go in the MetricsService as it should be reusable for all counters
https://github.com/hawkular/hawkular/blob/ee54b476d14c039f5bb5df908ecab44ce547ae31/console/src/main/scripts/plugins/metrics/ts/services/metricsService.ts#L82-82
Also, can you add a retrieveCounterMetrics method.
This can be in a separate commit.